File: app/src/main/java/com/example/experience/AdminMainActivity.java
package com.example.experience;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.view.MenuItem;

import com.example.experience.Fragments.BibliotekaFragment;
import com.example.experience.Fragments.CitatyFragment;
import com.example.experience.Fragments.HomeFragment;
import com.example.experience.Fragments.ProfilFragment;
import com.google.android.material.bottomnavigation.BottomNavigationView;

public class AdminMainActivity extends AppCompatActivity {
    private BottomNavigationView bottomNavigationView;
    @SuppressLint("MissingInflatedId")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_admin_main);
        bottomNavigationView = findViewById(R.id.navigation);
        bottomNavigationView.setOnNavigationItemSelectedListener(navListener);
        getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container,
                new HomeFragment()).commit();
    }
    private BottomNavigationView.OnNavigationItemSelectedListener navListener =
            new BottomNavigationView.OnNavigationItemSelectedListener() {
                @Override
                public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                    Fragment selectedFragment = null;

                    switch (item.getItemId()) {
                        case R.id.glavnaya:
                            selectedFragment = new HomeFragment();
                            break;
                        case R.id.biblioteka:
                            selectedFragment = new BibliotekaFragment();
                            break;
                        case R.id.citaty:
                            selectedFragment = new CitatyFragment();
                            break;
                        case R.id.profil:
                            selectedFragment = new ProfilFragment();
                            break;
                    }

                    getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container,
                            selectedFragment).commit();

                    return true;
                }
            };
}

File: app/src/main/java/kz/orkenniet/admin/presentation/AdminMainActivity.kt
package kz.orkenniet.admin.presentation

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import by.kirich1409.viewbindingdelegate.viewBinding
import com.google.android.material.bottomnavigation.BottomNavigationView
import kz.orkenniet.R
import kz.orkenniet.databinding.ActivityAdminMainBinding
import kz.orkenniet.home.presentation.HomeFragment
import kz.orkenniet.library.presentation.LibraryFragment
import kz.orkenniet.profile.presentation.ProfileFragment
import kz.orkenniet.rating.presentation.RatingFragment

class AdminMainActivity : AppCompatActivity(R.layout.activity_admin_main) {

    private val binding: ActivityAdminMainBinding by viewBinding(ActivityAdminMainBinding::bind)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding.navigation.setOnNavigationItemSelectedListener(navListener)
        supportFragmentManager.beginTransaction().replace(
            R.id.fragment_container,
            HomeFragment()
        ).commit()
    }

    private val navListener = BottomNavigationView.OnNavigationItemSelectedListener { item ->
        var selectedFragment = when (item.itemId) {
            R.id.glavnaya -> HomeFragment()
            R.id.biblioteka -> LibraryFragment()
            R.id.citaty -> RatingFragment()
            R.id.profil -> ProfileFragment()
            else -> null
        }
        supportFragmentManager.beginTransaction().replace(
            R.id.fragment_container,
            selectedFragment!!
        ).commit()
        true
    }
}
